src/main/java/org/veupathdb/service/eda/ds/plugin/abundance/AbundanceBoxplotPlugin.java

```diff
@@ -133,8 +134,8 @@ protected void writeResults(OutputStream out, Map<String, InputStream> dataStrea
       String command = "plot.data::box(vizData, map, '" +
           spec.getPoints().getValue() + "', " +
           showMean + ", " + 
-          computeStats + ", " + 
-          showMissingness + ", " +
+          computeStats + ", '" + 
+          deprecatedShowMissingness + "'', " +
```

Comment:
only 1 quote needed on this line i think

Reply:
also noticed this in the other plugins

Reply:
lol i think my text editor played me a trick